File: M3-Programacion/UF1/Practica11-M03/Exercici1_P11.java
import java.io.*;
public class Exercici1_P11 {
	public static void main (String[] args) throws IOException  {
		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
		int posicion;
		String texto;
		char caracter;
		
		System.out.print("Pon un string: ");
		texto= reader.readLine();
		System.out.print("Pon un caracter: "); 
		caracter = reader.readLine().charAt(0); 
		
		posicion = texto.indexOf(caracter); //Devuelve la posición del caracter
		
		if(posicion >= 0){ //Si la posicion devuelta es mayor a 0 es que lo ha encontrado
			System.out.println("El caracter "+caracter+" aparece en "+texto);
		}else{
			System.out.println("El caracter "+caracter+" no aparece en "+texto);
		}
	}
}

File: M3-Programacion/UF1/Practica11-M03/Exercici1_P11.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

texto = input("Pon un string: ")
caracter = input("Pon un caracter: ")

"""
if(caracter in texto): #Si el caracter esta en la string
	print("El caracter",caracter,"aparece en",texto)
else:
	print("El caracter",caracter,"no aparece en",texto)

"""
longitud = len(texto)

i=0
encontrado=0

while(i < longitud):
	if(texto[i] == caracter):
		i = i+1
		encontrado=encontrado+1;
	else:
		i = i+1


if(encontrado >= 0):
	print("El caracter",caracter,"aparece en",texto)
else:
	print("El caracter",caracter,"no aparece en",texto)
